compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/BLangClassDefinition.java

```diff
@@ -55,6 +55,7 @@
     public List<BLangSimpleVariable> referencedFields;
     public int precedence;
     public boolean isServiceDecl;
+    public boolean isObjectConstructor;
```

Comment:
We already have an `OBJECT_CTOR` flag, right? Can't we reuse the same without adding another field?

Reply:
Oh totally missed that, will use it